tests/test_foundry.py

```diff
@@ -24,8 +27,60 @@
     "short_name": "iris_example",
     "package_type": "tabular"
 }
-# Globus endpoint for '_test_blaiszik_foundry_iris_v2.1'
-test_data_source = "https://app.globus.org/file-manager?origin_id=e38ee745-6d04-11e5-ba46-22000b92c6ec&origin_path=%2Ffoundry%2F"
+
+
+test_metadata = {
+    "keys":[
+        {
+            "key": ["sepal length (cm)"],
+            "type": "input",
+            "units": "cm",
+            "description": "sepal length in unit(cm)"
+        },
+        {
+            "key": ["sepal width (cm)"],
+            "type": "input",
+            "units": "cm",
+            "description": "sepal width in unit(cm)"
+        },
+        {
+            "key": ["petal length (cm)"],
+            "type": "input",
+            "units": "cm",
+            "description": "petal length in unit(cm)"
+        },
+        {
+            "key": ["petal width (cm)"],
+            "type": "input",
+            "units": "cm",
+            "description": "petal width in unit(cm)"
+        },
+        {
+            "key": ["flower type"],
+            "type": "output",
+            "units": "",
+            "description": "flower type"
+        }
+    ],
+    'splits': [
+        {'label': 'train', 'path': 'train.json', 'type': 'train'},
+        {'label': 'test', 'path': 'test.json', 'type': 'test'}
+    ],
+    #"short_name": "example_AS_iris_test_{:.0f}".format(timestamp),
+    "data_type": "tabular",
+    'task_type': ['unsupervised', 'generative'],
+    'domain': ['materials science', 'chemistry'],
+    'n_items': 1000
+}
+# Globus endpoint for '_iris_dev'
+test_data_source = "https://app.globus.org/file-manager?origin_id=e38ee745-6d04-11e5-ba46-22000b92c6ec&origin_path=%2Ffoundry-test%2F"
+
+#Quick function to delete any downloaded test data
+def delete_test_data(f):
```

Comment:
great, I would just rename this to `_delete_test_data()` so the user knows it's meant to be a private function.

also, could you make `f` a little more informative? unclear if it takes a folder path or a filename